xbmc/cores/VideoPlayer/DVDInputStreams/InputStreamAddon.cpp

```diff
@@ -68,6 +68,15 @@ CInputStreamAddon::~CInputStreamAddon()
 
 bool CInputStreamAddon::Supports(BinaryAddonBasePtr& addonBase, const CFileItem &fileitem)
 {
+  // Error for users to show deprecation, can be removed in Kodi 20
```

Comment:
Maybe add a doxygen `@deprecated` or even a `@todo` to make this easier to track and completely remove in v20?

Reply:
Makes sense, maybe both.

Reply:
Or just the @todo. I think the deprecation is for function calls.